package.json

```diff
@@ -87,15 +87,13 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-sort-imports-es6-autofix": "^0.5.0",
-    "focus-visible": "^5.2.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "jest-circus": "^24.9.0",
     "jest-static-stubs": "^0.0.1",
     "lerna": "^3.20.2",
     "lodash": "^4.17.21",
     "node-sass": "^4.12.0",
-    "normalize.css": "^8.0.1",
```

Comment:
These aren't needed in the workspace anymore, since the Container package encapsulates these.